### DISCUSSION TITLE
Is TT deleting my previous seasons?

### DISCUSSION THREAD

**Discussion by eleete** [jessedp/tablo-tools-electron#206] [Q&A] [answered]
I have a folder on my hard drive called Tablo Recordings.
I exported all the recordings from my Tablo attached usb drive into the folder on the hard drive, Everything went well.
I had multiple seasons for several shows and multiple files within them.
Couple days later, I export again and I have all the new recordings but TT seems to have deleted the previous seasons as if I was starting over.
Is that what it was supposed to do? I was hoping to build a library of recordings, but it seems I might have to use a sync tool and just have a Tablo Dump folder that (one way) syncs with a permanent Library folder.
My Tablo usb is only 256gb my hard drive is 16TB so I was thinking I could just build a library on the 16TB every few days of recording by bulk exporting.
Am I on the right track with that thinking?  Create a dump folder, then move all the new recordings over to the Library folder on the 16TB?

**Comment by dougK44883** [comment]
I've just finished exporting recording via Tablo Tools, and no previous files were deleted.  I presume you've confirmed the option to overwrite/skip if a file already exists?  I'm not sure of the particulars, I've notice recent versions "clean-up" or arbitrarily delete partially exported files if you cancel or the process gets interpreted.  (in the past is was the user to decide weather or not to delete files from their drive).

I'm not exactly sure what your issue, there's little detail what you're actually doing or any setting/configuration.  I too export most all my recordings for archival/library into multiple folders by various naming schemes.  Never had a bulk delete of file of exported files with any of the available 3rd party export apps.  Closest Tablo Tools, is to overwrite an existing file - if the new recording is incomplete, the "good" recording would be lost (presuming you already had a successful recording in the existing file)

**Reply by eleete** [reply]
I'll edit this if I remember any other juicy details but...
I am telling it to skip files that exist, and to delete once exported (I assume that means off the Tablo)
I went to the shopping cart looking icon, top center. Told it to select all recordings, then pointed to a single folder on my hard drive.
It downloaded the files and created the folders with show names, then inside those folders S01, S02... inside those each episode files.
Couple of days later I did the exact same thing but now the hard drive folder had the structure in place, with the files in them.
After it completed,
I went into one of the show folders, that had 3 seasons previously only now it only had one season (which was the new recording) and the others were gone.
I'm going to test again and play some more, I just found it odd.
If I find or think of more details, I'll come back and edit this.
Thanks for taking the time to read and respond.

**Comment by dougK44883** [comment, marked as answer]
It's very **unlikely** Tablo Tools is deleting your files.  I do nearly the say thing - export recording into specific folders, come back and export more recordings into the same already created folders, maybe new folders - an always the content of my folders remain unchanged.

Tablo Tools does have some logging, always check logs.  Confirm you are using the same structure and haven't "misplaced" the files.

I'm not aware of any facility to delete any files stored on your PC, other than overwriting an existing recording.... but then something is still there.  Tablo Tools doesn't really have a "file manager".
